### PR TITLE
Minor SplitChunks Documentation Clean Up

### PR DESCRIPTION
**Pull request by jacobangel**
Originally, i was just trying to change defer => differ, but I ended up doing a bit more editing.

### CODE REVIEW

File: src/content/plugins/split-chunks-plugin.md [hunk 1]
```diff
@@ -33,9 +34,9 @@ When trying to fulfill the last two conditions, bigger chunks are preferred.
 
 ## Configuration
 
-For developers that want to have more control over this functionality, webpack provides a set of options to better fit your needs. If you're changing the configuration, it's a good idea to measure the impact of your changes to ensure there's a real benefit.
+Webpack provides a set of options for developers that want more control over this functionality.
```
Comment (jacobangel) [inline src/content/plugins/split-chunks-plugin.md]: webpack should be lower case

**Comment by jacobangel** [comment]
Oof, my apologies. I think by updating with master I must have reset the review.

**Comment by montogeek** [comment]
@jacobangel No, What do you mean with reset the review?

**Comment by jacobangel** [comment]
I meant I merged master, and my "approval" disappeared.

File: src/content/plugins/split-chunks-plugin.md [hunk 2]
```diff
@@ -94,7 +95,7 @@ module.exports = {
 };
 ```
 
-Alternatively, you can provide a function for more control. The return value will indicate whether to include each chunk.
+Alternatively, you may provide a function for more control. The return value will indicate whether to include each chunk.
```
Comment (EugeneHlushko) [inline src/content/plugins/split-chunks-plugin.md]: is this really a thing?

File: src/content/plugins/split-chunks-plugin.md [hunk 3]
```diff
@@ -18,9 +19,9 @@ Since webpack v4, the `CommonsChunkPlugin` was removed in favor of `optimization
 
 ## Defaults
 
-Out of the box `SplitChunksPlugin` should work great for most users.
+Out of the box `SplitChunksPlugin` should work well for most users.
```
Comment (EugeneHlushko) [inline src/content/plugins/split-chunks-plugin.md]: feels like it didnt work great for most users and we had to change to to `well` 😕

Comment (jacobangel) [reply src/content/plugins/split-chunks-plugin.md]: In this case, "well" reads as the more natural adverb in this sentence.

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**